docs/user-guide/lns-module-configuration.md

```diff
@@ -0,0 +1,44 @@
+| LOG_TO_CONSOLE              | Indicates whether logging to console is enabled or not                                 | No (default to `true`)              |
+| LOG_TO_TCP                  | Indicates whether logging to TCP is enabled or not (used mainly for integration tests) | No (defaults to `false`)            |
+| LOG_TO_HUB                  | Indicates whether logging to IoT Hub is enabled or not                                 | No (defaults to `false`)            |
+| LOG_TO_TCP_ADDRESS          | IP address for TCP logs                                                                | Yes, only if TCP logging is enabled |
+| LOG_TO_TCP_PORT             | TCP port to send logs to                                                               | Yes, only if TCP logging is enabled |
+| NETID                       | Gateway network ID                                                                     | No (defaults to network id 1)       |
+| AllowedDevAddresses         | List of allowed dev addresses from which message processing is enabled.                | No (by default allows all messages coming from the defined `NETID` to be processed) |
+| LNS_SERVER_PFX_PATH         | Path of the .pfx certificate to be used for LNS Server endpoint                        | No                                  |
+| LNS_SERVER_PFX_PASSWORD     | Password of the .pfx certificate to be used for LNS Server endpoint                    | No                                  |
+| CLIENT_CERTIFICATE_MODE     | Specifies the client certificate mode with which the server should be run              | No (defaults to `NoCertificate`)    |
+| LNS_VERSION                 | Version of the LNS                                                                     | No                                  |
+| IOTHUB_CONNECTION_POOL_SIZE | AMQP Connection Pool Size for communication to IoT Edge Hub / IoT Hub (depending on `ENABLE_GATEWAY`). Increasing this value to higher number will improve scalability; for more information see [Scalability](./scalability.md) | No (defaults to 1) |
+| APPINSIGHTS_INSTRUMENTATIONKEY | Instrumentation key for forwarding metrics to Application Insights                  | No                                  |
+
+The following settings can be configured via desired properties of the Network
+Server module twin in IoT Hub:
+
+| Property name                 | Description                                                                                                                                   | Mandatory               |
+| ----------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------- |
+| FacadeServerUrl               | Azure Facade function url                                                                                                                     | Yes                     |
```

Comment:
@danigian the function url (and the same for auth code) is essentially set both [through env variables](https://github.com/Azure/iotedge-lorawan-starterkit/blob/4b2cdf9d425c55ee0c67b5900fa8c9ff211c9206/LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/NetworkServerConfiguration.cs#L160) and [through the desired properties of the twin](https://github.com/Azure/iotedge-lorawan-starterkit/blob/4b2cdf9d425c55ee0c67b5900fa8c9ff211c9206/LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/ModuleConnection/ModuleConnectionHost.cs#L199). Do you know why this is the case?

Reply:
I think this is in order to be able to run locally, we take this value from the env vars in this case